libcodechecker/libhandlers/cmd.py

```diff
@@ -594,6 +594,15 @@ def add_arguments_to_parser(parser):
     results.set_defaults(func=cmd_line_client.handle_list_results)
     __add_common_arguments(results, has_matrix_output=True)
 
+    results = subcommands.add_parser(
+        'statistics',
```

Comment:
Why is this a new subcommand instead of `CodeChecker cmd sum` redesigned to give this output?

@gyorb